packages/contact-center/store/src/store.types.ts

```diff
@@ -0,0 +1,31 @@
+import {AgentLogin, IContactCenter, Profile, Team} from '@webex/plugin-cc';
+
+interface WithWebex {
+    webex: { cc: IContactCenter };
+}
+
+interface WithWebexConfigAndToken {
+    webexConfig: any; // Replace 'any' with the actual type of webexConfig
+    access_token: string;
+}
+  
+type InitParams = WithWebex | WithWebexConfigAndToken;
+
+interface IStore {
+    teams: Team[];
+    loginOptions: string[];
```

Comment:
I'd be more happy to get this from the SDK rather than typing here as it would be subject to change. Leaving it as it is for now

CC - @Kesari3008 @rsarika

Reply:
This array will change based on what is received from the agent config so it makes sense to keep it as a string array